packages/p2p/src/components/dp2p-blocked/dp2p-blocked-description.jsx

```diff
@@ -7,31 +7,20 @@ import { Localize } from '../i18next';
 const Dp2pBlockedDescription = () => {
     const { general_store } = useStores();
 
-    let blocked_description;
-
-    if (general_store.is_high_risk_fully_authed_without_fa) {
+    const getBlockedDescription = () => {
         if (general_store.is_p2p_blocked_for_pa) {
-            blocked_description = (
+            return (
                 <Localize i18n_default_text='P2P transactions are locked. This feature is not available for payment agents.' />
             );
-        } else {
-            blocked_description = (
-                <Localize i18n_default_text='To enable this feature you must complete the following:' />
-            );
+        } else if (general_store.is_high_risk) {
+            return <Localize i18n_default_text='To enable this feature you must complete the following:' />;
         }
-    } else if (general_store.is_p2p_blocked_for_pa) {
-        blocked_description = (
-            <Localize i18n_default_text='P2P transactions are locked. This feature is not available for payment agents.' />
-        );
-    } else {
-        blocked_description = (
-            <Localize i18n_default_text='Please use live chat to contact our Customer Support team for help.' />
-        );
-    }
+        return <Localize i18n_default_text='Please use live chat to contact our Customer Support team for help.' />;
+    };
```

Comment:
Would this be possible, not 100% sure for `Localize` but there's some in the codebase that uses variables:
```
const getBlockedDescription = () => {
   if (general_store.is_p2p_blocked_for_pa) {
      return 'P2P transactions are locked. This feature is not available for payment agents.'
   } else if (general_store.is_high_risk) {
      return 'To enable this feature you must complete the following:'
   }
   return 'Please use live chat to contact our Customer Support team for help.'
}

...
return (
  ...
   <Localize i18n_default_text={getBlockedDescription()} />
)
```
cc: @carol-binary